packages/api/src/brevo-api/brevo-api-contact.service.ts

```diff
@@ -101,13 +108,18 @@ export class BrevoApiContactsService {
         }
     }
 
-    public async findContact(idOrEmail: string | number, scope: EmailCampaignScopeInterface): Promise<BrevoContactInterface> {
+    public async findContact(idOrEmail: string | number, scope: EmailCampaignScopeInterface): Promise<BrevoContactInterface | undefined> {
```

Comment:
Optional:
In my opinion null would be more appropriate in this case. It's how nest also handles it when you check in the database if an entry exists. It return null.

Undefined is in my opinion more like "unassigned", in this case "null" would mean the lookup for the contact was made, but there was nothing.

But maybe that's just me and we don't have a best practice as far as I know, so both is fine for me.

Reply:
[44f4fbb](https://github.com/vivid-planet/comet-brevo-module/pull/108/commits/44f4fbb035e3503331a86614a2207463105bd03e)